src/splitInteger.test.js

```diff
@@ -19,3 +19,10 @@ test('should sort parts ascending if they are not equal', () => {
 test('should add zeros if value < numberOfParts', () => {
```

Comment:
The test case 'should add zeros if value < numberOfParts' is missing its implementation.